src/Initialization.jl

```diff
@@ -0,0 +1,112 @@
+"""
+    module Initialization
+In light of the increasing complexity of the ReadSave module, I aim to move the functions relating to actually initialising simulations in one place, as well as some dependencies relating to equilibration cycles.
+"""
+
+module Initialization
+
+export init_sim,restart_ptmc, initialisation
+
+using StaticArrays,DelimitedFiles,Random
+using ..MCStates
+using ..BoundaryConditions
+using ..Configurations
+using ..InputParams
+using ..MCMoves
+using ..EnergyEvaluation
+using ..Exchange
+using ..ReadSave
+
+
+"""
+    init_sim(pot ;dir=pwd())
+Function designed to take an input file named by kwarg `file` and open and initalise the states and parameters required for the simulation. 
+    
+    ---This is not exactly the same as the restart as the params etc are distributed in separate files by the checkpoint function to prevent duplication and slowdown throughout the simulation. 
+"""
+function init_sim(pot ,file,eq_percentage)
```

Comment:
rfile and eq_percentage should be optional keyword arguments? As described in the docu string